src/moby/output.go

```diff
@@ -27,53 +27,57 @@ const (
 
 var outFuns = map[string]func(string, io.Reader, int) error{
 	"kernel+initrd": func(base string, image io.Reader, size int) error {
-		kernel, initrd, cmdline, err := tarToInitrd(image)
+		kernel, initrd, cmdline, ucode, err := tarToInitrd(image)
 		if err != nil {
 			return fmt.Errorf("Error converting to initrd: %v", err)
 		}
-		err = outputKernelInitrd(base, kernel, initrd, cmdline)
+		// TODO: Handle ucode
```

Comment:
This is handled in this case, it will already be prepended to the initrd, so I dont think you need a TODO.

Reply:
crap, was left over. Removed now